massgov/mayflower-patternlab
# DP- Add sanitization of text in map pin title

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

I'm not sure if the issue of Map pins having HTML in the title should be on the Drupal side of things in providing plain text for the titles, but regardless I added a tiny JS function to sanitize the title to be free of HTML tags. May still want to check in on the Drupal side of things just to make sure things are as expected. 

## Related Issue / Ticket

- [JIRA issue](https://jira.state.ma.us/browse/DP-7761)

## Steps to Test


This issue isn't present in Mayflower, only the integration with Drupal, but to mock this issue:
1. Add HTML to a `name` within a `infoWindow` in `google-map.json` i.e: `Department of Conservation and <a href="">Recreation<a>`
2. Ensure that when you view `http://localhost:3000/?p=molecules-google-map` and hover over that map pin, the title says `Department of Conservation and Recreation` (no HTML)

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
